tests/conftest.py

```diff
@@ -65,23 +65,3 @@ async def fixture_async_session() -> AsyncGenerator[AsyncSession, None]:
         await test_connexion.run_sync(Base.metadata.drop_all)
 
     await engine.dispose()
-
-
-@pytest.fixture(autouse=True)
-def caplog(caplog: LogCaptureFixture):  # pylint: disable=redefined-outer-name
-    """
-    Make pytest work with loguru. See:
-    https://loguru.readthedocs.io/en/stable/resources/migration.html#making-things-work-with-pytest-and-caplog
-
-    :param caplog: pytest fixture
-    :return: loguru compatible caplog
-    """
-    handler_id = logger.add(
-        caplog.handler,
-        format="{message}",
-        level=0,
-        filter=lambda record: record["level"].no >= caplog.handler.level,
-        enqueue=True,
-    )
-    yield caplog
-    logger.remove(handler_id)
```

Comment:
@Paldana99 Why did you disable the fixture ?

Reply:
It cause all test to fail. Due that now the config is made directly in the svp_harvester

Reply:
@Paldana99 What if we need caplog ? https://loguru.readthedocs.io/en/stable/resources/migration.html#replacing-caplog-fixture-from-pytest-library

Reply:
Now is working properly, i restore the fixture. And added a test with a  caplog assert.